src/utils/fetch-graphql.js

```diff
@@ -80,13 +86,19 @@ const handleGraphQLErrors = async ({
   response,
   errorMap,
   panicOnError,
-  reporter,
+  structuredReporter,
   errorContext,
 }) => {
   const pluginOptions = getPluginOptions()
 
   if (!response) {
-    reporter.panic(response)
+    /**
+     * FIXME
+     *
+     * 1. We're logging an undefined variable here instead of a message.
+     * 2. What scenarios lead us to this point? Can we assign a code?
+     */
+    structuredReporter.panic(response)
```

Comment:
cc @TylerBarnes any thoughts here?

Reply:
Uhh 🤔 wth hahaha. This is definitely an accident.. This is what happens when you don't have code review 🤦 .

Reply:
I'll fix this separately from this PR. Feel free to leave your FIXME in there

Reply:
The fact that there's also a return after it panics is a big LOL